examples/processor/reverse/reverse.go

```diff
@@ -0,0 +1,93 @@
+/*
+http://www.apache.org/licenses/LICENSE-2.0.txt
+
+Copyright 2016 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package reverse
+
+import (
+	"strconv"
+
+	"github.com/intelsdi-x/snap-plugin-lib-go/v1/plugin"
+)
+
+// RProcessor test processor
+type RProcessor struct {
+}
+
+// Process test process function
+func (r RProcessor) Process(mts []plugin.Metric, cfg plugin.Config) ([]plugin.Metric, error) {
+	metrics := []plugin.Metric{}
+	for _, m := range mts {
+		switch m.Data.(type) {
+		case int:
+			m.Data = stringToInt(reverse(intToString(m.Data.(int))))
+		case int32:
+			i32 := int(m.Data.(int32))
+			m.Data = stringToInt(reverse(intToString(i32)))
+		case int64:
```

Comment:
it makes sense to add all built-in types. Only that I thought this is a fake one I didn't add all of them.